src/components/organisms/Dashboard.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Pane, Select, Text } from 'evergreen-ui';
+import AddComponentButton from '../atoms/AddComponentButton';
+import DashboardItem from '../molecules/DashboardItem';
+import { useReactiveVar } from '@apollo/client';
+import { dashboardItemsVar } from '../../cache/cache';
+import DataWrapper from '../molecules/DataWrapper';
+import ThresholdChart from '../atoms/ThresholdChart';
+import { ParentSize } from '@visx/responsive';
+
+const Dashboard: React.FC = () => {
+    //Apollo local state
+    const dashboardItems = useReactiveVar(dashboardItemsVar);
```

Comment:
Due to this, you should wrap your storybook template in a provider as well.

@blauks knows more about this (regarding templates and exports).

Reply:
Done